<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsINDI</class>
 <widget class="QWidget" name="OpsINDI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>485</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="generalGroup">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="telescopeLabel">
          <property name="toolTip">
           <string>Default INDI telescope port</string>
          </property>
          <property name="text">
           <string>Telescope port:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KLineEdit" name="kcfg_telescopePort">
          <property name="text">
           <string>/dev/ttyUSB0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="telescopeLabel_2">
          <property name="toolTip">
           <string>Default INDI telescope port</string>
          </property>
          <property name="text">
           <string>Focuser port:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="KLineEdit" name="kcfg_focuserPort">
          <property name="text">
           <string>/dev/ttyUSB1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="videoLabel">
          <property name="toolTip">
           <string>Default INDI video port</string>
          </property>
          <property name="text">
           <string>Video port:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KLineEdit" name="kcfg_videoPort">
          <property name="text">
           <string>/dev/video0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="telescopeLabel_3">
          <property name="toolTip">
           <string>Default INDI telescope port</string>
          </property>
          <property name="text">
           <string>Filter port:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="KLineEdit" name="kcfg_filterPort">
          <property name="text">
           <string>/dev/ttyUSB2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="videoLabel_2">
          <property name="toolTip">
           <string>Default INDI video port</string>
          </property>
          <property name="text">
           <string>INDI Server:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KLineEdit" name="kcfg_indiServer">
          <property name="text">
           <string>/usr/bin/indiserver</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="videoLabel_3">
          <property name="toolTip">
           <string>Default INDI video port</string>
          </property>
          <property name="text">
           <string>INDI Drivers Directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="kcfg_indiDriversDir">
          <property name="text">
           <string>/usr/share/indi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectDriversDirB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="fitsLabel">
          <property name="whatsThis">
           <string>Specify where new FITS images are saved once captured and downloaded</string>
          </property>
          <property name="text">
           <string>Default FITS directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="kcfg_fitsDir"/>
        </item>
        <item>
         <widget class="QPushButton" name="selectFITSDirB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="autoGroup">
       <property name="title">
        <string>Automatic Device Updates</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QCheckBox" name="kcfg_useTimeUpdate">
          <property name="toolTip">
           <string>Update telescope clock upon connection</string>
          </property>
          <property name="whatsThis">
           <string>Synchronize KStars date and time automatically from the update source.</string>
          </property>
          <property name="text">
           <string>Time</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_useGeographicUpdate">
          <property name="toolTip">
           <string>Update telescope geographical location upon connection</string>
          </property>
          <property name="whatsThis">
           <string>Synchronize KStars location settings from the update source</string>
          </property>
          <property name="text">
           <string>&amp;Geographic location</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
       <zorder>kcfg_useTimeUpdate</zorder>
       <zorder>kcfg_useGeographicUpdate</zorder>
       <zorder>verticalSpacer_2</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="displayGroup">
       <property name="title">
        <string>Display</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="kcfg_showTargetCrosshair">
          <property name="toolTip">
           <string>Display the telescope position on the sky map</string>
          </property>
          <property name="whatsThis">
           <string>Display a telescope marker on the sky map in order to track the telescope motion across the sky.</string>
          </property>
          <property name="text">
           <string>&amp;Device target crosshair</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_showINDIMessages">
          <property name="toolTip">
           <string>Display INDI status messages in the status bar</string>
          </property>
          <property name="whatsThis">
           <string>Display incoming INDI status messages in KStars status bar</string>
          </property>
          <property name="text">
           <string>INDI messages in status &amp;bar</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_showFITS">
          <property name="whatsThis">
           <string>When a FITS image is captured from a camera, download and display the FITS automatically in KStars FITS Viewer</string>
          </property>
          <property name="text">
           <string>&amp;Automatic display of FITS</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="whatsThis">
      <string>Synchronize KStars according to either the computer or device time and location settings.</string>
     </property>
     <property name="title">
      <string>Update Source</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="kcfg_useComputerSource">
        <property name="text">
         <string>Computer</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kcfg_useDeviceSource">
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>429</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="autoGroup_2">
       <property name="toolTip">
        <string>Assign color values for your filter wheel slots</string>
       </property>
       <property name="whatsThis">
        <string>Assign an alias to each filter wheel slot number. For example, you can assing slot 0 to Red, slot 1 to Green, etc..</string>
       </property>
       <property name="title">
        <string>Filter Wheel</string>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>11</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1">
            <property name="text">
             <string>Slot:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel2">
            <property name="text">
             <string>Alias:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="KComboBox" name="filterSlotCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="KLineEdit" name="kcfg_filterAlias">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="whatsThis">
        <string>Specify range of ports that INDI Server will bind to when starting new drivers</string>
       </property>
       <property name="title">
        <string>Server Port</string>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>From:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>To:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="KLineEdit" name="kcfg_serverPortStart"/>
          </item>
          <item>
           <widget class="KLineEdit" name="kcfg_serverPortEnd"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_useTimeUpdate</tabstop>
  <tabstop>kcfg_useGeographicUpdate</tabstop>
  <tabstop>kcfg_showTargetCrosshair</tabstop>
  <tabstop>kcfg_showINDIMessages</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
